src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -62,28 +68,29 @@ private void registeredCommands() {
         SmartDashboard.putData("Rocket Hatch Mid Auto", autoMoveLift(Heights.kRocketHatchMid));
         SmartDashboard.putData("Rocket Hatch High Auto", autoMoveLift(Heights.kRocketHatchHigh));
 
+        SmartDashboard.putData("Fold Arms", foldArms());
+
     }
 
-    // urgay
     public enum Heights {
         // Loading Station 19"
-        kLoadingStation(7.2),
+        kLoadingStation(5.8),
```

Comment:
In the future we definitely want to store these types of values in `Inches` and set the conversion in the robot map. That way if we change the hardware we don't have to change these values. We can work on this when we meet.

Reply:
ignoring now